src/components/game/controller.js

```diff
@@ -1,5 +1,6 @@
-export const calculateWinner = (squares) => {
-  const lines = [
+export const calculateWinner = (currentlyPlayedMoves) => {
+  const possibleWinningCombinations = [
+    // a, b, c - positions in the array for a winning combinations in the array
```

Comment:
The name of this variable describes the comment well enough for the comment to be removed.